sentry_sdk/utils.py

```diff
@@ -897,36 +898,17 @@ def exceptions_from_error_tuple(
     # type: (...) -> List[Dict[str, Any]]
     exc_type, exc_value, tb = exc_info
 
-    is_exception_group = BaseExceptionGroup is not None and isinstance(
-        exc_value, BaseExceptionGroup
+    _, exceptions = exceptions_from_error(
```

Comment:
[question] does the `exceptions_from_error` function already internally handle the non-exception group case?

From naively looking at the diff, it seems that the code for handling exceptions which are not exception groups has been deleted, so I am just wondering where this logic is handled now.

Reply:
I have now updated the code to make it easier to grasp what is happening. (The diff is very hard to read, just checkout the file)

The function `exceptions_from_error` handles all the cases, single exceptions, chained exceptions and exception groups. It walks through the complete tree of exceptions. 

(We can in another PR also think about removing `walk_exception_chain` everywhere and just use `exceptions_from_error` for everything. But this is outside of the scope of this PR)